pkg/splunk/handler.go

```diff
@@ -20,10 +20,18 @@ const (
 // SplunkHandler sends records to a Splunk instance as events.
 type SplunkHandler struct {
 	level  slog.Level
-	splunk *splunkLogger
+	splunk eventer
 	jh     slog.Handler
 }
 
+// TODO might be unused
+type eventer interface {
+	event(b []byte) (int, error)
+	flush()
+	close()
+	statistics() Stats
+}
```

Comment:
I will remove this, found a way how to test this handler without an interface and a mock.